workos/directory_sync.py

```diff
@@ -111,12 +111,19 @@ def get_group(self, group):
         )
 
     def list_directories(
-        self, domain=None, search=None, limit=RESPONSE_LIMIT, before=None, after=None
+        self,
+        domain=None,
+        organization=None,
```

Comment:
It looks like `organization` is still where it was before, the formatting just changed 🤔

Reply:
Oh man I made the change in the example app where i was testing things out, not the sdk. 🤦🏽   This is fixed now!